packages/web/src/components/organisms/TransactionForm/index.tsx

```diff
@@ -27,7 +27,11 @@ const Wrap = styled.div`
 `
 
 export const TransactionForm = ({ propertyAddress }: Props) => {
-  const [isCancelCompleted, setIsCancelCompleted] = useState(false)
+  const [withdrawable, setWithdrawable] = useState(false)
+  const [isCountingBlocks, setIsCountingBlocks] = useState(false)
+  const [remainBlocks, setRemainBlocks] = useState(0)
+  const [timer, setTimer] = useState<NodeJS.Timeout>()
```

Comment:
The number of states used in the Component is increasing. We need to refactor with custom hooks in the future...

- `withdrawable` manages the completion of the *CANCEL*. This value is used to control the button. 
- `isCountingBlocks` manages the launching of the timer
- remainBlocks is the value of `withdrawStatus - blockNumber`
- timer manages setInterval function